tools/pipelines/templates/include-install-pnpm.yml

```diff
@@ -44,7 +44,8 @@ steps:
     condition: or(succeeded(), canceled())
     # workspace-concurrency 0 means use use the CPU core count. This is better than the default (4) for larger agents.
     script: |
-      corepack enable
-      pnpm -v
+      echo "Using node $(node --version)"
+      sudo corepack enable
```

Comment:
Why is this needed?

Reply:
@tylerbutler because of https://dev.azure.com/fluidframework/public/_build/results?buildId=174472&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=996a989a-edd9-5084-d9de-d81350de1bfc